app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
```

Comment:
- [ ] Please make sure your design matches that provided to you in the wireframe as mentioned in the project requirements:
![image](https://github.com/Swarzstein/blog-app-2/assets/45460192/18c7f699-3547-4c3c-903e-86f8bdfb8ec9)
### Your Design
![image](https://github.com/Swarzstein/blog-app-2/assets/45460192/4c1f54af-582c-4595-935f-93a645afc3a5)
### Required
![image](https://github.com/Swarzstein/blog-app-2/assets/45460192/983b14db-102b-4725-aa97-215b705cc524)

### **NB:** Do it for the sign up page as well

Reply:
These views are for a phone screen

Reply:
Hello @Swarzstein . Yes, they are and the requirement is to follow that wireframe. It is also mentioned in our code review guidelines that your UI should resemble that in the wireframe.